server/routes/journeys/record-decision/confirmation/tests.cy.ts

```diff
@@ -50,6 +50,36 @@ context('test /record-decision/confirmation', () => {
         nextSteps: `<script>alert('xss-nextSteps');</script>`,
         actionOther: `<script>alert('xss-actionOther');</script>`,
       },
+      csipRecord: {
+        status: 'PLAN_PENDING',
```

Comment:
The test now expects this to be whatever status is set in the cypress stub. You might be able to get away with removing the whole thing? `cy.task('stubCsipRecordGetSuccess')` should cover this

Reply:
Good point, done!